.github/workflows/mpet-docker-test.yml

```diff
@@ -9,9 +9,18 @@ jobs:
     
     runs-on: ubuntu-latest
     container:
-      image: vikko00/mpet-deps:latest
+      image: vikko00/daetools:1.9.0
 
     steps:
+    - name: update container
+      run: |
+        apt-get update
+        apt-get upgrade -y
+
+    - name: Install mpet additional dependencies
+      run: |
+        pip install numpy scipy matplotlib pyQt5 h5py coverage coveralls pytest sympy
+
```

Comment:
How about installing dependencies by running setup.py? This would help check for missing dependencies in the install script. 'pytest' still needs to be installed manually I think.

Reply:
@v1kko, I found a typeo in setup.py. It's now possible to install all mpet and testing dependencies with the command `pip install .[test]`.

Reply:
That is much better, I agree